clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -3504,6 +3512,35 @@ private ListTransactionsResponse createListTransactionsResponse() {
         return new ListTransactionsResponse(response);
     }
 
+    private GetTelemetrySubscriptionsRequest createGetTelemetrySubscriptionsRequest(short version) {
+        return new GetTelemetrySubscriptionsRequest.Builder(new GetTelemetrySubscriptionsRequestData()
+            .setClientInstanceId(Uuid.randomUuid())
+        ).build(version);
+    }
+
+    private GetTelemetrySubscriptionsResponse createGetTelemetrySubscriptionsResponse() {
+        GetTelemetrySubscriptionsResponseData response = new GetTelemetrySubscriptionsResponseData();
+        response.setClientInstanceId(Uuid.randomUuid());
```

Comment:
Wouldn't it make sense to set error code and throttle time here too?

Reply:
Added them, additional setters too.